desktop/libs/notebook/src/notebook/connectors/flink_sql.py

```diff
@@ -70,13 +72,20 @@ def __init__(self, user, interpreter=None):
 
     self.options = interpreter['options']
     api_url = self.options['url']
+    self.default_catalog = self.options[DEFAULT_CATALOG_PARAM] if DEFAULT_CATALOG_PARAM in self.options else None
+    self.default_database = self.options[DEFAULT_DATABASE_PARAM] if DEFAULT_DATABASE_PARAM in self.options else None
```

Comment:
Nit: Maybe try `self.options.get(DEFAULT_CATALOG_PARAM)` and `self.options.get(DEFAULT_DATABASE_PARAM)` ?